src/puppetlabs/puppetdb/http/reports.clj

```diff
@@ -2,31 +2,35 @@
   (:require [puppetlabs.puppetdb.query.paging :as paging]
             [puppetlabs.puppetdb.query-eng :refer [produce-streaming-body]]
             [net.cgrand.moustache :refer [app]]
+            [puppetlabs.puppetdb.http.query :as http-q]
+            [puppetlabs.puppetdb.http.events :as e]
             [puppetlabs.puppetdb.middleware :refer [verify-accepts-json validate-query-params
                                                     wrap-with-paging-options]]))
 
 (defn query-app
   [version]
   (app
-    [&]
+    [hash "events" &]
```

Comment:
I have another ticket to create for doing this, again, didn't want to add any more into it.